src/data/files/file.model.ts

```diff
@@ -188,33 +194,42 @@ FileSchema.plugin(AutoIncrement, {
   inc_field: 'fileId',
 });
 
+FileSchema.plugin(mongoosePaginate);
+
 export type FileMongooseDocument = mongoose.Document & DbFile;
```

Comment:
This type def I used here is wrong, should be:
`export type FileMongooseDocument = mongoose.Document<DbFile>;`